R/pkg/tests/fulltests/test_eager_execution.R

```diff
@@ -21,11 +21,7 @@ context("Show SparkDataFrame when eager execution is enabled.")
 
 test_that("eager execution is not enabled", {
   # Start Spark session without eager execution enabled
-  sparkSession <- if (windows_with_hadoop()) {
-    sparkR.session(master = sparkRTestMaster)
-  } else {
-    sparkR.session(master = sparkRTestMaster, enableHiveSupport = FALSE)
-  }
+  sparkR.session(master = sparkRTestMaster)
```

Comment:
you should definitely set `enableHiveSupport = FALSE` - historically this hasn't work well in other R tests when hive catalog is enabled

Reply:
as mentioned here https://github.com/apache/spark/pull/22455#discussion_r220030686

Reply:
Sure. Guess I misinterpreted your other comment.